sqlglotrs/src/lib.rs

```diff
@@ -13,8 +13,10 @@ pub use self::tokenizer::Tokenizer;
 #[derive(Debug)]
 #[pyclass]
 pub struct Token {
-    #[pyo3(get)]
+    #[pyo3(get, name = "token_type_index")]
```

Comment:
so this is token type index?

Reply:
it is, yes. But the rust code base doesn't need to worry about this detail. Do you suggest renaming the attribute?

Reply:
it’s ok